py_zipkin/logging_helper.py

```diff
@@ -58,23 +49,18 @@ def __init__(
         self.sa_endpoint = None
 
     def start(self):
-        """Actions to be taken before request is handled.
-        1) Attach `zipkin_logger` to :class:`ZipkinLoggerHandler` object.
-        2) Record the start timestamp.
-        """
-        zipkin_logger.removeHandler(null_handler)
-        zipkin_logger.addHandler(self.log_handler)
+        """Actions to be taken before request is handled."""
+
+        # Record the start timestamp.
         self.start_timestamp = time.time()
         return self
 
     def stop(self):
         """Actions to be taken post request handling.
-        1) Log the service annotations to scribe.
-        2) Detach `zipkin_logger` handler.
         """
+
+        # Log the service annotations to scribe.
```

Comment:
Removed it. I also renamed the function in `emit_spans`, since we cannot know what the transport handler will do. Maybe instead of logging the spans it'll use the HTTP api.